compute/gpu/how-to/use-preinstalled-env.mdx

```diff
@@ -6,11 +6,11 @@ content:
   h1: How to use the preinstalled environment
   paragraph: This page explains how to use the preinstalled environment
 dates:
-  validation: 2021-12-14
-  posted: 2021-12-14
+  validation: 2022-03-25
+  posted: 2022-03-25
 ---
 
-Once you've launched one of our ready-made [Docker images](/compute/gpu/how-to/use-gpu-with-docker) on your Scaleway GPU Instance, you have access to a pre-installed Python environment. A number of useful AI core packages and tools are installed, including scipy, numpy, scikit-learn, jupyter, tensorflow and the Scaleway SDK. Depending on the Docker image you choose, other packages and tools will also be pre-installed, providing a convenient framework environment for you so that you can begin work immediately. For the full list of Scaleway's AI Docker images and their pre-installations, see [TODO INSERT LINK](link).
+Once you've [launched](/compute/gpu/how-to/use-gpu-with-docker) one of our ready-made [Docker images](/compute/gpu/reference-content/docker-images) on your Scaleway GPU Instance, you have access to a pre-installed Python environment. A number of useful AI core packages and tools are installed, including scipy, numpy, scikit-learn, jupyter, tensorflow and the Scaleway SDK. Depending on the Docker image you choose, other packages and tools will also be pre-installed, providing a convenient framework environment for you so that you can begin work immediately.
```

Comment:
```suggestion
Once you have [launched](/compute/gpu/how-to/use-gpu-with-docker) one of our ready-made [Docker images](/compute/gpu/reference-content/docker-images) on your Scaleway GPU Instance, you have access to a pre-installed Python environment. A number of useful AI core packages and tools are installed, including `scipy`, `numpy`, `scikit-learn`, `jupyter`, `tensorflow` and the Scaleway SDK. Depending on the Docker image you choose, other packages and tools will also be pre-installed, providing a convenient framework environment for you so that you can begin work immediately.
```